deluge/log.py

```diff
@@ -107,7 +107,7 @@ def findCaller(self):  # NOQA
 }
 
 
-def setup_logger(level="error", filename=None, filemode="w"):
+def setup_logger(level="error", filename=None, filemode="w", max_size="50M"):
```

Comment:
I know but who would need to rotate logs at less than 1MB? I still maintain that you should not have human readable format as a input for an internal method, it should all be converted upon user input.